fjord/l10n/management/commands/extract.py

```diff
@@ -41,6 +44,11 @@ def create_pounit(filename, lineno, message, comments):
         unit.addnote(comment, 'developer')
 
     unit.addlocation('%s:%s' % (filename, lineno))
+    # FIXME: Add variable format flags
+    # if python-format:
+    #     unit.settypecomment('python-format', present=True)
+    # if python-brace-format:
+    #     unit.settypecomment('python-brace-format', present=True)
```

Comment:
xgettext (and thus "./mange.py makemessages") adds variable format flags. "./manage.py extract" does not so far.

I have no idea if any of the translations tools we have make use of them, but it'd let me fix some issues with dennis that I'm grappling with now if dennis knew exactly what variable formats were used in a given string.